# PAEA

<p align="center">
  <img width="250" height="250" src="https://github.com/AliYoussef96/PEAA/blob/main/logos/paealogo.png">
</p>


**If you find PAEA useful, please cite our paper:**


## Description

PAEA (Pathway Association Enrichment Analysis), is a new method for metabolomics pathway enrichment analysis using a novel association algorithm to increase the initial input list with significant associates metabolites (compounds), to give a bigger picture on possible activated pathways. In this version KEGG (Kyoto Encyclopedia of Genes and Genomes) is used as background database, for enrishment and association analysis. All the species available on KEGG can be used by the user.

For more information about the PAEA implementation and how the algorithm work, please take a look on the original [manuscript]() publish at [journal name]().

## Availability and Installation 


**PAEA is available in two forms on R;**

### 1- Offline verision as a Shiny app.

#### Step 1. Install package dependencies 

To use PAEA 1.0.0, first install all package dependencies. Ensure that you have necessary system environment configured. R base with version > 3.6.1 is recommended.

```R




```

**See the Full Documentation for all the [details]() ; how to use the PAEA Shiny app. and how to save the results.**

2-	Online version as a Shiny app.

PAEA can also run online at; 

## Package information

- Version: **v1.0.0**
- License: GPL-3
- Encoding: UTF-8


# Contribution Guidelines

For bugs and suggestions, the most effective way is by raising an issue on the github issue tracker. Github allows you to classify your issues so that we know if it is a bug report, feature request or feedback to the authors.

**Email: Proteomicslab2017@gmail.com**
**Email: ali.anwar@57357.org**
**Email: aliali.mostafa99@gmail.com**

# Future plans

# Citation

![](https://www.57357.org/app/uploads/2019/12/logo-2.png)
